### PR TITLE
Add Alpine Install Types to JSON

### PR DESCRIPTION
**Pull request by newzealandpaul**
## Description

Modifies JSON files for scripts that can include alpine. 

I would appreciate if a couple of people can approve and test.

## Type of change
Please check the relevant option(s):

- [x] Bug fix (non-breaking change that resolves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (a fix or feature that would cause existing functionality to change unexpectedly)
- [ ] New script (a fully functional and thoroughly tested script or set of scripts.)
- [ ] Self-review performed (I have reviewed my code, ensuring it follows established patterns and conventions)
- [ ] Documentation update required (this change requires an update to the documentation)

### CODE REVIEW

File: json/docker.json [hunk 1]
```diff
@@ -24,6 +24,17 @@
                 "os": "debian",
                 "version": "12"
             }
+        },
+        {
+            "type": "alpine",
+            "script": "alpine-docker.sh",
```
Comment (havardthom) [inline json/docker.json:30]:
```suggestion
            "script": "ct/alpine-docker.sh",
```

File: json/grafana.json [hunk 2]
```diff
@@ -24,6 +24,17 @@
                 "os": "debian",
                 "version": "12"
             }
+        },
+        {
+            "type": "alpine",
+            "script": "alpine-grafana.sh",
```
Comment (havardthom) [inline json/grafana.json:30]:
```suggestion
            "script": "ct/alpine-grafana.sh",
```

File: json/nextcloudpi.json [hunk 3]
```diff
@@ -24,6 +24,17 @@
                 "os": "debian",
                 "version": "12"
             }
+        },
+        {
+            "type": "alpine",
+            "script": "alpine-nextcloud.sh",
```
Comment (havardthom) [inline json/nextcloudpi.json:30]:
```suggestion
            "script": "ct/alpine-nextcloud.sh",
```

File: json/vaultwarden.json [hunk 4]
```diff
@@ -24,6 +24,17 @@
                 "os": "debian",
                 "version": "12"
             }
+        },
+        {
+            "type": "alpine",
+            "script": "alpine-vaultwarden.sh",
```
Comment (havardthom) [inline json/vaultwarden.json:30]:
```suggestion
            "script": "ct/alpine-vaultwarden.sh",
```

File: json/zigbee2mqtt.json [hunk 5]
```diff
@@ -24,6 +24,17 @@
                 "os": "debian",
                 "version": "12"
             }
+        },
+        {
+            "type": "alpine",
+            "script": "alpine-zigbee2mqtt.sh",
```
Comment (havardthom) [inline json/zigbee2mqtt.json:30]:
```suggestion
            "script": "ct/alpine-zigbee2mqtt.sh",
```

**Review by havardthom — APPROVED** [review]

**Comment by newzealandpaul** [comment]
Thanks for noticing those mistakes folks. I am pleased to retire the script generator!

**Pull request merged**